mapping/project-level-identification.csv

```diff
@@ -4,5 +4,5 @@ Sector,"Develop a list of sectors relevant to country e.g. housing, transport, e
 Subsector,"Develop a subset for each sector e.g. Transport could be subdivided into national highway, local road, railway, port, airport etc.",Project Level: Publish in ``additionalClassifications``,"OCDS: Copy from ``planning.project.additionalClassifications`` (Budget and projects extension, PPP profile)",`Issue #16 <https://github.com/open-contracting/infrastructure/issues/16>`_
 Project name,Specify the project name,Project Level: Publish as ``title``,"OCDS: Copy from ``planning.project.title`` (Budget and projects extension, PPP profile); if not available, infer from ``tender.title``",
 Project Location,Briefly specify location of the project,"Project Level: Publish using the ``locations`` fields as an address, geometry (point/line/polygon) or gazetteer entry","OCDS: Copy from ``planning.project.locations`` (Budget and projects extension, PPP profile); if not available, infer from ``tender.items.deliveryLocation`` or ``tender.items.deliveryAddress`` (Location extension)",
-Purpose,Specify the socio economic purpose of the project,,OCDS: Infer from ``planning.rationale`` and check ``planning.documents`` for documents with ``.type`` set to 'needsAssessment',`Issue #15 <https://github.com/open-contracting/infrastructure/issues/15>`_
+Purpose,Specify the socio economic purpose of the project,"Project Level: Publish a free text description in ``purpose`` and if classified against a local or global codelist then publish in ``additionalClassifications``.",OCDS: Infer from ``planning.rationale`` and check ``planning.documents`` for documents with ``.type`` set to 'needsAssessment',`Issue #15 <https://github.com/open-contracting/infrastructure/issues/15>`_
```

Comment:
Prefer:

> Project Level: Publish in ``purpose``; if classified against a codelist, also publish in ``additionalClassifications``.

"local or global" covers all options, so cut those words for brevity/clarity. We don't have "a free text description" for similar fields; I suggest omitting here as well. Use semi-colon "if" construction as in other mappings, which is easier to read.